src/util.py

```diff
@@ -146,18 +146,24 @@ def get_latest_gv_version(gv_major_version, channel):
         name += "-" + channel
     # A-C builds against geckoview-omni
     # See https://github.com/mozilla-mobile/android-components/commit/0b349f48c91a50bb7b4ffbf40c6c122ed18142d3
+    # However, geckoview-omni requires exoplayer2 which comes from the lite build, so check for that too
+    name_lite = name
     name += "-omni"
```

Comment:
Note to self: I didn't remember what Python does with strings and the `+=` operator. I tested`name_lite` keeps the old value.